lib/trento/activity_logging/registry.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Trento.ActivityLogging.Registry do
+  @moduledoc """
+  Activity logging registry
+  """
+
+  @type logged_activity :: {controller :: module(), action :: atom()}
+
+  @login_attempt {TrentoWeb.SessionController, :create}
+  @api_key_generation {TrentoWeb.V1.SettingsController, :update_api_key_settings}
+  @saving_suma_settings {TrentoWeb.V1.SUMACredentialsController, :create}
```

Comment:
Maybe for later: Use the OpenAPI spec as a single source of truth for this purpose.

Reply:
Interesting idea. Let's indeed keep that on top of our minds as dealing with openapispex schemas would be a beast on its own :smile: